manifests/facts.pp

```diff
@@ -27,10 +27,14 @@
 
   # Puppet Enterprise supplies its own ruby version if your using it.
   # A modern ruby version is required to run the executable fact
+  # Since PE 2015.2 the agents are All in One (AIO) wich also include it's own ruby version
   if $::puppetversion =~ /Puppet Enterprise/ {
     $ruby_bin = '/opt/puppet/bin/ruby'
     $dir      = 'puppetlabs/'
-  } else {
+  } elsif $::aio_agent_version =~ /1\.3\.*[0-9]*/ {
```

Comment:
I hesitated on this one.
But I guess that It's a safe to assume that Ruby will be included in an "All In One" version of a puppet agent...